ddtrace/settings/config.py

```diff
@@ -355,9 +605,11 @@ def _add(self, integration, settings, merge=True):
             # >>> config._add('requests', dict(split_by_domain=False))
             # >>> config.requests['split_by_domain']
             # True
-            self._config[integration] = IntegrationConfig(self, integration, _deepmerge(existing, settings))
+            self._integration_configs[integration] = IntegrationConfig(
```

Comment:
Is supporting `IntegrationConfig` out of scope?

Reply:
yes